models.py

```diff
@@ -108,7 +106,7 @@ def run(self):
 
     def countdown(self, exc_time):
         for sec in range(exc_time):
-            self.logger.info("{0} seconds until the next exercise".format(exc_time-sec))
+            self.logger.debug("{0} seconds until the next exercise".format(exc_time-sec))
```

Comment:
lol mostly because the logs would fill up my console screen, but I understand how this could be useful since it kinda acts like a timer.

Thoughts on changing it to a one line print statement that overwrites itself? (see http://stackoverflow.com/questions/4897359/output-to-the-same-line-overwriting-previous-output-python-2-5)